srcpkgs/itch/template

```diff
@@ -0,0 +1,61 @@
+# Template file for 'itch'
+pkgname=itch
+version=1.18.0
+revision=1
+archs="i686 x86_64"
+build_style=go
+hostmakedepends="wget"
+makedepends="curl"
```

Comment:
Why does it need curl has makedep? This sounds wrong.

Reply:
I will remove curl, I relied on other distros that use curl to create the package.